admin/create_admin.py

```
@@ -0,0 +1,152 @@
+                continue
+
+            break
+
+    # Get role
+    role = args.role
+
+    try:
+        # Create the user
+        user_id = auth_manager.create_admin_user(username=username, password=password, email=email, role=role)
+
+        print(f"✅ Successfully created admin user:")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)